.gitpod.dockerfile

```diff
@@ -1 +1,7 @@
-FROM gitpod/workspace-mysql
\ No newline at end of file
+FROM gitpod/workspace-full
+
+# Install Redis.
+RUN sudo apt-get update \
```

Comment:
We usually don't need `sudo` for the Dockerfile cmds :thinking:

Reply:
Actually I don't know this, I found this in gitpod docs.